specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json

```diff
@@ -271,37 +272,7 @@
             "schema": {
               "$ref": "#/definitions/ComponentPurgeResponse"
             }
-          }
-        },
-        "x-ms-examples": {
-          "ComponentPurge": {
-            "$ref": "./examples/ComponentsPurge.json"
-          }
-        }
-      }
-    },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Insights/components/{resourceName}/operations/{purgeId}": {
```

Comment:
Removing this operation is a breaking change, you can see the existing Go package that is broken: https://github.com/Azure/azure-sdk-for-go/pull/1505/files#diff-a2a9bdbadfa2cabd9d88ec1c345f0a53L259